### PR TITLE
Pull in changes from OEP Dev Branch

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

File: SEED Benchmark/OEI/classes/pollseedforproperties.xml [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/mule/ee/tracking/current/mule-tracking-ee.xsd
+http://www.mulesoft.org/schema/mule/scripting http://www.mulesoft.org/schema/mule/scripting/current/mule-scripting.xsd
+http://www.mulesoft.org/schema/mule/sfdc http://www.mulesoft.org/schema/mule/sfdc/current/mule-sfdc.xsd
+http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
 http://www.mulesoft.org/schema/mule/schedulers http://www.mulesoft.org/schema/mule/schedulers/current/mule-schedulers.xsd">
 
     <flow name="pollseedforpropertiesFlow">
         <poll doc:name="Poll">
-            <schedulers:cron-scheduler expression="${cron.timer}"/>
+            <fixed-frequency-scheduler frequency="10" timeUnit="MINUTES"/>
```
Comment (nllong) [inline SEED Benchmark/OEI/classes/pollseedforproperties.xml]: @jasbrand It looks like this is hard coded now. Is this as expected?

File: SEED Benchmark/OEI/classes/pollseedforproperties.xml [hunk 2]
```diff
@@ -41,50 +41,57 @@ http://www.mulesoft.org/schema/mule/schedulers http://www.mulesoft.org/schema/mu
...
+        <file:outbound-endpoint path="/tmp" outputPattern="lastReadDate.txt" responseTimeout="10000" doc:name="File"/>
         <catch-exception-strategy doc:name="Catch Exception Strategy" doc:description="Exception handling for the C2M listener">
             <set-variable variableName="ErrorLog" value="#[flowVars.ErrorLog + &quot;\n \n [Info1] &quot; + exception.getEvent() + &quot;with the following details: \n&quot; + exception.getCauseException()]" doc:name="Variable: Update Error Log"/>
             <flow-ref name="ErrorHandler" doc:name="ErrorHandler"/>
         </catch-exception-strategy>
     </flow>
     <flow name="getpropertiesfororg" processingStrategy="synchronous">
-        <http:request config-ref="SEED_API" path="/labels/filter/?inventory_type=property&amp;organization_id=#[flowVars.organization_id]" method="POST" doc:name="HTTP: Get all labels for my organization"/>
+        <http:request config-ref="SEED_API" path="/labels/filter/?inventory_type=property_view&amp;organization_id=#[flowVars.organization_id]" method="POST" doc:name="HTTP: Get all labels for my organization">
```
Comment (nllong) [inline SEED Benchmark/OEI/classes/pollseedforproperties.xml]: thanks for updating the property_view!

File: SEED Benchmark/mulesoft/src/main/app/pollseedforproperties.xml [hunk 3]
```diff
@@ -2,23 +2,22 @@
...
+http://www.mulesoft.org/schema/mule/scripting http://www.mulesoft.org/schema/mule/scripting/current/mule-scripting.xsd
+http://www.mulesoft.org/schema/mule/sfdc http://www.mulesoft.org/schema/mule/sfdc/current/mule-sfdc.xsd
+http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
 http://www.mulesoft.org/schema/mule/schedulers http://www.mulesoft.org/schema/mule/schedulers/current/mule-schedulers.xsd">
 
     <flow name="pollseedforpropertiesFlow">
         <poll doc:name="Poll">
-
-            <schedulers:cron-scheduler expression="${cron.timer}"/>
+            <fixed-frequency-scheduler frequency="10" timeUnit="MINUTES"/>
```
Comment (nllong) [inline SEED Benchmark/mulesoft/src/main/app/pollseedforproperties.xml]: hard coded?

**Comment by nllong** [comment]
After merging, merge up to Dev branch.

File: SEED Benchmark/OEI/classes/pollseedforproperties.xml (hunk 1 above)
Comment (jasbrand) [reply SEED Benchmark/OEI/classes/pollseedforproperties.xml]: I think so. It doesn't change so we can leave it. This way the timer will start immediately

File: SEED Benchmark/mulesoft/src/main/app/pollseedforproperties.xml (hunk 3 above)
Comment (jasbrand) [reply SEED Benchmark/mulesoft/src/main/app/pollseedforproperties.xml]: @nllong same as above. Same timer.

File: SEED Benchmark/OEI/classes/pollseedforproperties.xml (hunk 1 above)
Comment (nllong) [reply SEED Benchmark/OEI/classes/pollseedforproperties.xml]: But weren’t we using the timer specified by the user? Doesn’t this remove that functionality?

Comment (jasbrand) [reply SEED Benchmark/OEI/classes/pollseedforproperties.xml]: This comes directly from the configuration file. The user can't affect it and it is static for each deployed instance but let's revert it. No sense in losing configurability

# Quartz expression to define the polling frequency to call SEED API for updated Benchmarks
# http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html
# To fire every hour:
#   0 0 0/1 ? * * *
# For debugging, use the command below to fire every minute
#   0 0/1 * ? * * *
cron.timer=${OEP_CRON_TIMER}

**Comment by nllong** [comment]
Exactly my concern. I’ll await your update.

Il giorno mar 3 set 2019 alle 19:32 Jeff Asbrand <notifications@github.com>
ha scritto:

**Comment by nllong** [comment]
Updated PR is here: #8

**Pull request closed without merging**